client/js/components/support/DpSupport.vue

```diff
@@ -15,11 +15,11 @@ All rights reserved
     <p class="u-mt-0_75">
       {{ Translator.trans('support.introduction') }}
     </p>
-    <p>
+    <p v-if="hasPermission()('feature_customer_support_technical_read')">
       {{ Translator.trans('support.contact.advice') }}
     </p>
     <h3
-      v-if="contacts.length > 0"
+      v-if="Object.keys(contacts).length > 0"
```

Comment:
why do you need this?

Reply:
Otherwise the condition was false regardless of number of contacts - it was fixed in main - but we cant test the renaming if the title is missing caused by the other bug.